source/includes/gho/_api.md

```diff
@@ -0,0 +1,305 @@
+# Queries
+## employee \([Employee](#employee)\)
+An Onboarding employee record
+
+Argument | Type | Description
+--------- | ----------- | -----------
+id | [Int](#int) | 
+## employees \([EmployeeConnection](#employeeconnection)\)
+A collection of Onboarding employee records
+
+Argument | Type | Description
+--------- | ----------- | -----------
+first | [Int](#int) | Returns the first _n_ elements from the list.
+after | [String](#string) | Returns the elements in the list that come after the specified global ID.
+last | [Int](#int) | Returns the last _n_ elements from the list.
+before | [String](#string) | Returns the elements in the list that come before the specified global ID.
+start_date | [DateFilter](#datefilter) | 
+date_of_birth | [DateFilter](#datefilter) | 
+with_custom_field_value | [WithCustomFieldValue](#withcustomfieldvalue) | 
+# Mutations
+## updateEmployeeProfile \([Employee](#employee)\)
+Update an Employee's profile
+
+Argument | Type | Description
+--------- | ----------- | -----------
+id | [ID](#id) | 
+employee_updates | [UpdateEmployee](#updateemployee) | 
+# Types
+## CustomField
+    Represents a single CustomField record for your company.  CustomFields can be stored and display in a variety of    different ways.  The types are described via the [CustomFieldTypes](#customfieldtypes) enum.  
+
+Field | Type | Description
+--------- | ----------- | -----------
+created_at | [DateTime](#datetime) | 
+field_type | [CustomFieldTypes](#customfieldtypes) | The field type determines how users input and view the data for this field.
+id | [ID](#id) | 
+name | [String](#string) | The name of this custom field as users would see it inside Greenhouse Onboarding.
+permanent_field_id | [String](#string) | A unique identifier for this CustomField.
+updated_at | [DateTime](#datetime) | 
+
+## CustomFieldValue
+A Custom Field Value Record
+
+Field | Type | Description
+--------- | ----------- | -----------
+created_at | [DateTime](#datetime) | 
+custom_field | [CustomField](#customfield) | 
+id | [ID](#id) | 
+updated_at | [DateTime](#datetime) | 
+value | [Value](#value) |     A different type of value will be stored based upon the field type of the [CustomField](#customfield).  Some types    have the data stored as a nested object.  Note that the type is a scalar named [Value](#value).  Even though    it appears to be an object, you are not able to use GraphQL to determine its shape.  
```

Comment:
Should you describe the value expected for each field type in that CustomFieldTypes data table?

Reply:
Drew's going to go into more detail here.